src/components/OrderCreateForm/OrderCreateForm.tsx

```diff
@@ -3,25 +3,18 @@ import { Button } from 'components/Button/Button';
 import { Error } from 'components/Error/Error';
 import { LoaderOverlay } from 'components/LoaderOverlay/LoaderOverlay';
 import { useOrder } from 'hooks/useOrder';
-import { ChangeEvent, FormEvent, ReactElement, useState } from 'react';
+import { FormEvent, ReactElement, useState } from 'react';
+import { OrderCreateDto } from 'dtos/OrderCreateDto';
 
 export const OrderCreateForm = (): ReactElement => {
-  const [address, setAddress] = useState('');
   const [validationMessage, setValidationMessage] = useState('');
   const { createOrder } = useOrder();
 
   const handleSubmit = (event: FormEvent<HTMLFormElement>): void => {
     event.preventDefault();
-    if (address) {
-      setValidationMessage('');
-      createOrder.mutate({ address });
-    } else {
-      setValidationMessage('Address is required to create an order.');
-    }
-  };
-
-  const handleChange = (event: ChangeEvent<HTMLTextAreaElement>): void => {
-    setAddress(event.target.value);
+    setValidationMessage('Unable to create order');
```

Comment:
Oh wait it already handles the error, ignore me - remove this, 
![image](https://user-images.githubusercontent.com/88429620/200847837-ecb24572-a673-462c-88e3-f096d3c281ed.png)